packages/apps-config/src/settings/endpoints.ts

```diff
@@ -160,6 +160,11 @@ function createTest (t: TFunction): LinkOption[] {
       text: t<string>('rpc.flamingfir', 'Flaming Fir (Substrate Testnet, hosted by Parity)', { ns: 'apps-config' }),
       value: 'wss://substrate-rpc.parity.io'
     },
+    {
```

Comment:
Small comment, just move this (Mashnet) down to below Mandala. (Alphabetical)

Reply:
Thanks, fixed in 560838e